src/assets/css/_images.scss

```diff
@@ -397,3 +405,7 @@
 .km-os-image-flatcar {
   @include mixins.background-image("/assets/images/os/flatcar.svg", 24px);
 }
+
+.with-text {
```

Comment:
I'd rather use something like we have for table column width, just for the text.

https://github.com/kubermatic/dashboard/blob/bfbb48dd3f1e0612d2ea8bc686d57cd721959d0f/src/assets/css/global/_main.scss#L277-L283